README.md

```diff
@@ -95,3 +86,20 @@ In dev, these are configured in the `.env` file.  In live, it's `heroku config`
 * `BUILD_SERVICE_HOST`: Hostname of the build service to use for fetching module metadata
 * `VIEW_CACHE_PATH`: Path on disk to use to cache view templates in Twig
 * `DEBUG_KEY`: String, if set in a `Debug` HTTP header, will set dev mode to true for that request only.
+
+### Orchestration files
+
+The following files are used in build, test and deploy automation:
+
+* `.dockerignore`: used to ignore things when adding files to the Docker image.  Generally this will be the same as the `.gitignore` file as the build happens at the container creation time.  See the `Dockerfile` for more info.
+* `app.json`: TODO
+* `Dockerfile`: TODO
```

Comment:
Ah yeh, I'm going to look into filling those in, but they were done before my time so I'm not entirely sure how to explain them. Might need some help with that.